recipes/openff-models/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "openff-models" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/openff-models/archive/{{ version }}.tar.gz
+  sha256: 0b7a8585c8c5235a0ab17faef4ebb731bd0538b148afc5375b2ce9b02282f555
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
Do you also need `openmm`?
https://github.com/openforcefield/openff-models/blob/main/openff/models/types.py#L9

The upstream package doesn't actually list it's `install_requirements` and the `__init__.py` in `openff.models` is basically a no-op, so it seems like other stuff could be missing.

Reply:
It should just be OpenMM; we track dependencies in our test environments (and subtract out those only needed for tests) since with feedstocks dependencies declared in the main repo is duplicate information and easy to fall out of date

install_requirements

Reply:
And thanks for catching the empty `__Init__.py` - we use a cookiecutter but need to make some modifications because we use namespace packages, and somehow the logic for setting `__version__` was not copied over correctly.